pootle/apps/pootle_app/migrations/0001_initial.py

```diff
@@ -28,6 +28,12 @@ class Migration(migrations.Migration):
             },
             bases=(models.Model, pootle.core.mixins.treeitem.CachedTreeItem),
         ),
+        migrations.RunSQL("ALTER TABLE `pootle_app_directory` ROW_FORMAT=DYNAMIC"),
```

Comment:
Where did the name `pootle_app_directory` come from? Not sure if I'm missing something. I see `db_table` defines the name in the second case, not sure what does that here. (These same two questions apply in a few of the cases, but I figure it's the same answer each time.)